ui-kit/src/components/container/container.js

```diff
@@ -0,0 +1,202 @@
+(function() {
+    const webcomponentsjs = 'https://cdnjs.cloudflare.com/ajax/libs/webcomponentsjs/0.7.23/webcomponents.min.js'
```

Comment:
I think the problem is this polyfill doesn't support `<slot>`.

Please see issues here...
https://github.com/webcomponents/webcomponentsjs/issues